regulations/static/regulations/js/source/views/sidebar/sidebar-view.js

```diff
@@ -34,10 +34,16 @@ var SidebarView = Backbone.View.extend({
         this.listenTo(this.externalEvents, 'breakaway:open', this.hideChildren);
 
         this.childViews = {};
+        this.viewlessChildren = {};  /* Children that don't have unique views */
```

Comment:
It's a great point -- I think we can get rid of that var altogether. I'll do so in #101